doc/source/tune.rst

```diff
@@ -30,6 +31,7 @@ Getting Started
 
     register_trainable("my_func", my_func)
 
+    ray.init()
     run_experiments({
```

Comment:
I believe you need to modify `rllib.rst` too

Reply:
Fixed